src/pages/shipping.js

```diff
@@ -1,9 +1,15 @@
-import { userAuthorization, removeActiveClass, getBoxes, getAllBoxes, getBoxesByLocation, hideAnimation, showAnimation, showNotifications, siteSpecificLocationToConceptId, locationConceptIDToLocationMap, addSelectionEventListener, searchSpecimenInstitute } from "./../shared.js";
-import { addEventAddSpecimenToBox, populateSpecimensList, addEventNavBarShipment, addEventNavBarBoxManifest, populateBoxManifestTable, populateBoxManifestHeader, populateSaveTable, populateShippingManifestBody, populateShippingManifestHeader, addEventNavBarShippingManifest, populateTrackingQuery, addEventSaveAndContinueButton, populateFinalCheck, populateBoxSelectList, addEventBoxSelectListChanged, addEventCompleteShippingButton, populateSelectLocationList, addEventChangeLocationSelect, addEventModalAddBox, populateTempNotification, addEventNavBarAssignTracking, addEventReturnToReviewShipmentContents, populateCourierBox, addEventSaveButton, addEventTrimTrackingNums, addEventCheckValidTrackInputs, addEventPreventTrackingConfirmPaste, addEventShipPrintManifest, addEventTrackingNumberScanAutoFocus } from "./../events.js";
+import { addBox, appState, conceptIdToSiteSpecificLocation, displayContactInformation, getAllBoxes, getLocationsInstitute, hideAnimation, locationConceptIDToLocationMap,
+        removeActiveClass, removeBag, removeMissingSpecimen, showAnimation, showNotifications, siteSpecificLocation, siteSpecificLocationToConceptId, sortBiospecimensList,
+        translateNumToType, userAuthorization } from "./../shared.js"
```

Comment:
should it be `"../shared.js"`

Reply:
That's nicer and more succinct - agree & updated style for equivalent functionality.